doc/nrf/protocols/matter/getting_started/advanced_kconfigs.rst

```diff
@@ -141,3 +141,11 @@ By default this Kconfig option is set to 1 second.
 .. note::
   The :kconfig:option:`CONFIG_CHIP_FACTORY_RESET_ERASE_NVS` Kconfig option is set to ``y`` by default.
   To disable removing application-specific non-volatile data when the :kconfig:option:`CONFIG_CHIP_LAST_FABRIC_REMOVED_ERASE_AND_REBOOT` Kconfig option is selected, set the :kconfig:option:`CONFIG_CHIP_FACTORY_RESET_ERASE_NVS` Kconfig option to ``n``.
+
+Read Client feature
```

Comment:
Super minor: How about `configuration`? `Feature` word seems a little bit off to me in this context. Using server or client is more like selecting a device role or configuration in Interaction Model. When I see feature I expect some bigger functionality that enables something extra besides basic protocol functionality. But yeah, maybe it's only me feeling such difference.

Reply:
We could use just "functionality".